config/manager/kustomization.yaml

```diff
@@ -7,5 +7,4 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: mongodb-atlas-controller
-  newTag: latest
+  newName: localhost:5000/mongodb-atlas-kubernetes-operator
```

Comment:
I believe this is noise from scaffolding and should be reverted.

Reply:
removed